### DISCUSSION TITLE
Buried [MAP RELEASE V1]

### DISCUSSION THREAD

**Discussion by TaborScott** [nzp-team/nzportable#854] [Map Releases]
_Are you afraid of dark, tight spaces? Confront your deepest fears as you battle the undead in an underground obstacle course of mental challenges. You are now 'Buried'._

[DOWNLOAD FOR WINDOWS](https://archive.org/download/buried_202407/Buried.zip)

Buried... Buried Buried Buried!

![Buried](https://github.com/user-attachments/assets/401cb5a6-3b37-48d9-92e9-a509d1c8a601)

The fan-favourite, obliteratingly fun map, set in Nevada or something... underground is scary okay? Don't die!

PLEASE NOTE: THIS MAP IS DESIGNED FOR SOLO PLAY, AND A GOOD COMPUTER IS NECESSARY IF YOU NEED NICE LOOKS!

This map is in a V1 playable state. There will be big and small changes in the future. If you want to have updates, follow this discussion! Also, due to the complexity of the map, zombies will get stuck! I am working on fixing it. For now, make sure to run around a lot!

Made By Scott. 

_one may be hidden, whilst the other watches from afar..._

**Comment by MotoLegacy** [comment]
You should add screenshots  as to not mislead people

**Reply by TaborScott** [reply]
Hi @MotoLegacy, I have the screenshots already updated but for some reason it has failed. I’ll reattempt in a little while!

**Comment by TaborScott** [comment]
HERE IS A FEW PICS AND A VIDEO!

[https://www.youtube.com/watch?v=Y0Io9rMOkWo&t=16s](url)

![OnePHOTO](https://github.com/user-attachments/assets/886336f3-0b02-45f1-bf21-cceaf358aae3)
![SECOND](https://github.com/user-attachments/assets/de704dbf-9215-4a82-82ff-e65a12450f79)

**Reply by Bonniegamer202** [reply]
looks pretty cool! some more details and it'd look like the orignal map.

**Reply by kmorris1077** [reply]
The youtube video is private.

**Comment by MotoLegacy** [comment]
Heads up! This does not meet install standard as the archive is not equivalent to the game directory root: https://github.com/nzp-team/nzportable/discussions/159

**Comment by tysonbulls64-hash** [comment]
pls optimize for ps vita
